internal/events/events.go

```diff
@@ -126,12 +127,23 @@ func eventString(writer io.Writer, event flow.Event) {
 }
 
 func printField(writer io.Writer, field cadence.Field, value cadence.Value) {
+	v := value.String()
 	var typeId string
+
+	defer func() {
+		if err := recover(); err != nil {
+			v = fmt.Sprintf("%s\n\t\thex: %x", v, v)
+			_, _ = fmt.Fprintf(writer, "\t\t- %s (%s): %s \n", field.Identifier, typeId, v)
+			log.Println("panic occurred:", err)
+		}
+	}()
```

Comment:
We don't need this recover if we handle optional type.
```suggestion
```

Reply:
addressed. Thanks!